server/src/api/taskratings/taskratings.routes.ts

```diff
@@ -16,20 +18,30 @@ taskratingRouter.get(
   requirePermission(Permission.TaskRatingRead),
   getTaskratings,
 );
+taskratingRouter.post(
+  '/taskrating',
```

Comment:
We could remove the singular versions of these routes. Or are these actually used somewhere?